randovania/games/fusion/gui/ui_files/preset_fusion_patches.ui

```diff
@@ -107,6 +122,20 @@
              </property>
             </widget>
            </item>
+           <item>
+            <widget class="QCheckBox" name="pb_without_bombs_check">
+             <property name="text">
+              <string>Enable Power Bombs without Bombs</string>
```

Comment:
I thought we didn't want to make this a setting due to PBs being locked by bombs not being interesting?